SNIPS/snip-86.md

```diff
@@ -0,0 +1,278 @@
+- revision: The Typed Data revision, '0' marks legacy Typed Data, while '1' indicates the active Typed Data standard.
+
+Message Object:
+
+- address: Specifies the wallet address responsible for verifying the user’s signature.
+- nonce: A one-time, unique number that makes each authentication request distinct to prevent replay attacks.
+- issuedAt: Timestamp indicating when the authentication request was generated, helping to establish its temporal validity.
+- statement: A user-readable message that the signer is agreeing to by providing their signature. This increases transparency and informs users about the purpose of their authentication.
+- uri: The identifier of the resource or the specific service that the authentication request is associated with, helping to contextualize the sign-in attempt.
+- version: The protocol version used in the message, ensuring compatibility between the user’s client and the dApp.
+- expirationTime (optional): The timestamp until which the authentication request remains valid, securing the process by limiting the time frame for a valid sign-in.
+- notBefore (optional): The earliest timestamp at which the authentication request can be considered valid, preventing its use before a certain time.
+
+### Sign-in Schema
+
+```json
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "properties": {
+    "domain": {
+      "type": "object",
+      "properties": {
+        "chainId": {
+          "type": "string",
+          "enum": ["SN_SEPOLIA", "SN_MAIN"],
+          "errorMessage": "ChainId must be one of 'SN_SEPOLIA', 'SN_MAIN'"
+        },
+        "name": {
+          "type": "string",
+          "errorMessage": "Name must be a string"
+        },
+        "version": {
+          "type": "string",
+          "maxLength": 31,
+          "pattern": "^\\d+\\.\\d+\\.\\d+$",
+          "errorMessage": "Version must be a string in the format x.y.z"
+        },
+        "revision": {
+          "type": "string",
+          "enum": ["0", "1"],
+          "errorMessage": "Revision must be a string and must be either '0' or '1'"
+        }
+      },
+      "required": ["chainId", "name", "version", "revision"],
+      "additionalProperties": false,
+      "errorMessage": "Domain must include chainId, name, version, revision"
+    },
+    "message": {
+      "type": "object",
+      "properties": {
+        "version": {
+          "type": "string",
+          "maxLength": 31,
+          "pattern": "^\\d+\\.\\d+\\.\\d+$",
+          "errorMessage": "Version must be a string in the format x.y.z"
+        },
+        "address": {
+          "type": "string",
+          "pattern": "^0x[a-fA-F0-9]{63,64}$",
+          "errorMessage": "Address must be a hexadecimal string with 66 characters, including the '0x' prefix"
+        },
+        "issuedAt": {
+          "type": "string",
+          "format": "date-time",
+          "errorMessage": "IssuedAt must be a valid date-time string"
```

Comment:
worth it to precise "ISO8601" too? To have similarity with EIP-4361

Reply:
I've included a "Message Fields" section to explain these fields in detail, similar to EIP-4361, looks good?